ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -175,6 +175,9 @@ public async Task<NotificationGroup> GetNotificationGroupAsync(int notificationI
                 .Where(n => n.NotificationId == notificationId)
                 .Select(n => n.Group)
                 .Include(g => g.Notifications)
+                    .ThenInclude(n => n.PatientDetails)
+                .Include(g => g.Notifications)
+                    .ThenInclude(n => n.Episode)
```

Comment:
Wonder why we need the expansion here..

Reply:
Sadly there's no .AlsoInclude or anything so need to do a separate Include for each ThenInclude